examples/path_requests_run.py

```diff
@@ -243,7 +252,7 @@ def correct_route_list(network, pathreqlist):
     # prepares the format of route list of nodes to be consistant
     # remove wrong names, remove endpoints
     # also correct source and destination
-    anytype = [n.uid for n in network.nodes() if not isinstance(n, Transceiver) and not isinstance(n, Fiber)]
+    anytype = [n.uid for n in network.nodes() if not isinstance(n, Transceiver)]
```

Comment:
restriction was placed on fiber type because when using excel input, autodesign create fiber names with syntax not explicit to the user, so that entering a node name as constraint may end up to a wrong interpretation eg
    aa -bb  -> create names such as 'fiber aa to bb', 'fiber bb to aa' .
    If user constraint is bb there is a ambiguity on the fiber direction.
However this is not a problem for user using json format. Since TransportPCE project needed such constraint, I preferred to removed this constraint now.
The user guide still warns excel user that only roadm names are correctly supported 
 a later PR solves this naming ambiguity (#278 )